src/content/1.3/code/ocaml/snippet01.ml

```diff
@@ -1,6 +1,6 @@
-module type Monoid = sig
-  type a
+module type MONOID = sig
```

Comment:
Is it an OCaml convention to capitalize the module type?

Reply:
Hi yes I believe so, e.g. see http://www.mseri.me/typeclass-ocaml/

> I will try to follow the ocaml conventions writing capitalized modules (e.g. Monoid) and fully capital signatures (e.g. MONOID).

Reply:
the convention originates from SML, it's loosely followed in OCaml really (even stdlib doesn't follow it).